config/dagu/scripts/git-maintenance.py

```
@@ -0,0 +1,256 @@
+
+    # Create check if it doesn't exist (idempotent)
+    create_payload = json.dumps({
+        "api_key": api_key,
+        "name": CHECK_NAME,
+        "timeout": 604800,
+        "grace": 86400,
+        "tags": "dagu",
+        "channels": "*",
+        "unique": ["name"],
+    })
+    subprocess.run(
```

Ruff (0.15.7)

[error] `subprocess` call: check for execution of untrusted input

(S603)